src/sqlfluff/core/errors.py

```diff
@@ -82,9 +82,9 @@ def pos_marker(self):
         """Get the position marker of the violation.
 
         Returns:
-            The :obj:`PosMarker` of the segments if the violation has a segment,
-            the :obj:`PosMarker` directly stored in a `pos` attribute or None
-            if neither a present.
+            The :obj:`FilePositionMarker` of the segments if the violation has a segment,
```

Comment:
Unrelated to the PR, but I think these comments were using an obsolete class name.